core/src/main/java/org/mskcc/cbio/portal/dao/DaoStructuralVariant.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) 2017 The Hyve B.V.
+ * This code is licensed under the GNU Affero General Public License (AGPL),
+ * version 3, or (at your option) any later version.
+ */
+
+/*
+ * This file is part of cBioPortal.
+ *
+ * cBioPortal is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+package org.mskcc.cbio.portal.dao;
+import org.mskcc.cbio.portal.model.StructuralVariant;
+import org.mskcc.cbio.portal.util.ProgressMonitor;
+
+import java.sql.*;
+import java.util.Set;
+
+public class DaoStructuralVariant {
+
+    private DaoStructuralVariant() {
+    }
+    /**
+     * Adds a new Structural variant record to the database.
+     * @param structuralVariant
+     * @return number of records successfully added
+     * @throws DaoException 
+     */
+    
+    public static long getLargestInternalId() throws DaoException {
```

Comment:
please remove this and use auto increment.

Reply:
Done.